### PR TITLE
DOCS: added description for bug template

### LINKED ISSUE #468: [Q]: Bug Template description change?
### **Description**

A brief description of the question or issue:
The current `Bug` issue description is not apt and descriptive.
Screenshot 2020-11-30 at 5 33 12 PM

Perhaps we can change it to something like- "Report a bug on this repo" 
(Also, perhaps we could replace that BUG template with something like: REPO-Issue:_report problems or suggestions for the repo_)

### Why is this essential?
The support repo is often like the entry gates to our community.
Often when some new member notice bugs/issues on the repo, it's like an opportunity to start contributing, but they might be confused due to the lacking issue description. 

### community info: 
You can also join the discord community [here](https://discord.com/invite/jZQs6Wu)
Feel free to check out other cool repositories of Eddie Jaoude Community [here](https://github.com/EddieJaoudeCommunity)

### PR DESCRIPTION
**Pull request by Vyvy-vi**
closes #468

#### What does this PR do?
adds a description to the Bug-template.
#### Description of Task to be completed?
NA
#### How can this be manually tested?
Yes, by copying the template to some test repo or at https://github.com/Vyvy-vi/Vyvy-vi/issues/new
#### Any background context you want to provide?
It would be better to have a description for this.
(might erase confusion while raising issues regarding the repo, rather than questions about the community)
#### What is the relevant issue link?
https://github.com/EddieJaoudeCommunity/support/issues/468
#### Questions:
Can this be better phrased?

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug.md [hunk 1]
```diff
@@ -1,15 +1,16 @@
 ---
 name: Bug
-about: Describe this issue template's purpose here.
-title: ''
-labels: bug
+about: Create a bug-report to help us address errors in the repo.
+title: '[BUG]'
+labels: 'bug'
```
Comment (dumbcoder7) [inline .github/ISSUE_TEMPLATE/bug.md:5]:
```suggestion
labels: 'bug'
```
I dont think we need this line to change. The labels on the bug template works perfectly fine.

**Review by dumbcoder7 — CHANGES_REQUESTED** [review]

**Review by dumbcoder7 — DISMISSED** [review]

**Comment by AllanRegush** [comment]
I'm not sure if we need [BUG] in the title if we have the `Bug` label. 

What do you think?

**Comment by AllanRegush** [comment]
Everything else looks good and adds value. 👍

**Comment by Vyvy-vi** [comment]
Hmm the label kinda beats the point

File: .github/ISSUE_TEMPLATE/bug.md [hunk 2]
```diff
@@ -10,6 +10,7 @@ assignees: ''
 **Description**
 
 A brief description of the question or issue, also include what you tried and what didn't work:
+<!--A clear and concise description of what the bug is.-->
```
Comment (nhcarrigan) [inline .github/ISSUE_TEMPLATE/bug.md:13]:
```suggestion
<!--A clear and concise description of what the bug is.-->
```

If we're using a comment for this, let's remove the text.

**Review by nhcarrigan — CHANGES_REQUESTED** [review]

**Review by dumbcoder7 — APPROVED** [review]

**Review by nhcarrigan — APPROVED** [review]
LGTM~! 🥳 

Thank you for your pull request! We look forward to additional contributions!

**Review by AllanRegush — APPROVED** [review]
Looks Good To Me 👍 Thank You

**Comment by eddiejaoude** [comment]
Thanks all 👍

**Pull request merged**